src/Templates/EnumTemplate.cshtml

```diff
@@ -13,10 +13,13 @@
 type @Model.Name string
 
 @EmptyLine
-const (
+@if (constants.Count() > 0)
```

Comment:
Changing this to `constants.Any()` will reduce this line from O(n) to O(1)

Reply:
Fixed